frappe/custom/doctype/custom_field/custom_field.json

```diff
@@ -153,10 +153,9 @@
   },
   {
    "default": "0",
-   "description": "If checked, this field will be not overwritten based on Fetch From if a value already exists.",
    "fieldname": "fetch_if_empty",
    "fieldtype": "Check",
-   "label": "Fetch If Empty"
+   "label": "Fetch on Save Only if Not Set"
   },
```

Comment:
“if empty“ actually seems more precise. It may be “set” to an empty string or set to an empty select option, but will still be fetched, right?

For checkboxes, this is awful either way. 🤷🏻‍♂️

Reply:
Again, trying to be consistent with the earlier label in DocType:
https://github.com/frappe/frappe/blob/6a22be0739c1aa4c2812cc5af442d1213fd5e83e/frappe/core/doctype/docfield/docfield.json#L249

FYI, the word "set" has the [most meanings in the dictionary](https://www.guinnessworldrecords.com/world-records/english-word-with-the-most-meanings). I'm sure there's one that means "empty" 😅

Reply:
LOL, that’s true, so whatever… 💁🏻‍♂️

Reply:
Maybe “Fetch on save if empty”?

<details>
<summary>But let others weigh in!*</summary>

*) IMO, there’s way too few discussion going on, particularly for such a complex and heterogenous endeavor, and particularly with so few style, UX and DX guides in place.

We may go and discuss general concepts to later derive individual answers, like in Roman Law. Or we may discuss individual PRs pars pro toto and infer general concepts from it, like in British Case Law. 

Both approaches work fine.
Neither one nor the other will either end up being chaotic or arbitrary, at least unstable, or may suggest the real discussion takes place somewhere else.
</details>

Reply:
So a part of the discussion did take place on Telegram, yes. But GitHub issues also serve as documentation and are a better place for this. I'll make a note of this, thanks!

Reply:
So I should really (and will) join the Telegram group!
But still, reporting results on Github is sooooo helpful, both atm and in the long term.
Thank you so much, I appreciate it! 🤗